tests/tensor/test_math_opt.py

```diff
@@ -4549,3 +4549,49 @@ def logit_fn(x):
     fg = optimize(FunctionGraph([x], [out]))
     assert not list(fg.toposort())
     assert fg.inputs[0] is fg.outputs[0]
+
+
+class TestDistributiveOpts:
+    x_at = vector("x")
+    y_at = vector("y")
+    a_at = matrix("a")
+
+    @pytest.mark.parametrize(
+        "orig_operation, optimized_operation",
+        [
+            (a_at * x_at + a_at * y_at, a_at * (x_at + y_at)),
```

Comment:
Small nitpick, the `_at` suffix makes it harder to read the tests

Reply:
They become extremely helpful when you're dealing with multiple variables of different types.  For instance, when using `kanren`, it's good to know that `x_lv` is the logic variable version of some term `x`, and that `x_at` is the Aesara tensor version.

Reply:
I'll just add the above said as a comment to make it easier to read/understand later on.